pkg/kubectl/cmd/resource/get.go

```diff
@@ -332,10 +332,14 @@ func (options *GetOptions) Run(f cmdutil.Factory, cmd *cobra.Command, args []str
 		return err
 	}
 	var sorter *kubectl.RuntimeSort
-	if options.Sort && len(objs) > 1 {
-		// TODO: questionable
-		if sorter, err = kubectl.SortObjects(cmdutil.InternalVersionDecoder(), objs, sorting); err != nil {
-			return err
+	if len(objs) > 1 {
+		if options.Sort {
+			// TODO: questionable
+			if sorter, err = kubectl.SortObjects(cmdutil.InternalVersionDecoder(), objs, sorting); err != nil {
+				return err
+			}
+		} else {
+			sorter, err = kubectl.SortObjects(cmdutil.InternalVersionDecoder(), objs, "{.lastTimestamp}")
```

Comment:
Not all the obj has this field `lastTimestamp`.

Reply:
> Flag --sort-by can help solve the order issue if necessary.

I know about --sort-by flag. This PR changes default sort order for 'kubectl events' output. It improves usability of the command. 

> Not all the obj has this field lastTimestamp.

For those objects that don't have it default order will simply not be changed.